Code/Editor/TrackView/AtomOutputFrameCapture.cpp

```diff
@@ -47,18 +48,39 @@ namespace TrackView
         AZ::Name viewName = AZ::Name("MainCamera");
         m_view = AZ::RPI::View::CreateView(viewName, AZ::RPI::View::UsageCamera);
         m_renderPipeline->SetDefaultView(m_view);
+        m_targetView = scene.GetDefaultRenderPipeline()->GetDefaultView();
     }
 
     void AtomOutputFrameCapture::DestroyPipeline(AZ::RPI::Scene& scene)
     {
+        AZ::Render::PostProcessFeatureProcessor* fp = scene.GetFeatureProcessor<AZ::Render::PostProcessFeatureProcessor>();
+        if (fp)
```

Comment:
nit/suggestion:

```
if (AZ::Render::PostProcessFeatureProcessor* fp = scene.GetFeatureProcessor<AZ::Render::PostProcessFeatureProcessor>())
```

This reduces the scope of `fp` so it can only be used while it's valid

Reply:
question: There isn't a corresponding `AddViewAlias` in `CreatePipeline` and I was just curious why this, would it make sense to add it there if it is currently added somewhere later on?

Reply:
That's a good point, i should add it there as well for consistency. It didnt seem necessary at first.